__tests__/app.test.js

```diff
@@ -95,6 +95,64 @@ describe("api/articles", () => {
         });
     });
   });
+  describe("PATCH /api/articles/:article_id", () => {
+    test("Status 200: Updates given articles votes and responds with the updated article", () => {
+      return request(app)
+        .patch("/api/articles/4")
+        .send({ inc_votes: -80 })
+        .expect(200)
+        .then(({ body }) => {
+          const article = body[0];
```

Comment:
It would be better if this article were taken out of the array before it's sent in the response, as only one article will ever be returned from this endpoint at a time